README.md

```diff
@@ -56,9 +56,9 @@ Let's start by looking at what a fully configured Store looks like. We will then
 ```kotlin
 StoreBuilder
     .from(
-        fetcher = nonFlowValueFetcher { api.fetchSubreddit(it, "10").data.children.map(::toPosts) },
-        sourceOfTruth = SourceOfTrue.from(
-            reader = db.postDao()::loadPosts,
+        fetcher = Fetcher.of { api.fetchSubreddit(it, "10").data.children.map(::toPosts) },
+        sourceOfTruth = SourceOfTrue.of(
+            flowReader = db.postDao()::loadPosts,
```

Comment:
I'm still not sure how discoverable it is to use parameter names do distinguish between the 2 functions. what happens if you call it without parameter names?

Reply:
without names this does work:

```
sourceOfTruth = SourceOfTruth.of(
                    db.postDao()::loadPosts,
                    db.postDao()::insertPosts,
                    db.postDao()::clearFeedBySubredditName,
                    db.postDao()::clearAllFeeds
                )
```

While this doesn't:

```
sourceOfTruth = SourceOfTruth.of(
                    {
                        db.postDao().loadPosts(it)
                    },
                    db.postDao()::insertPosts,
                    db.postDao()::clearFeedBySubredditName,
                    db.postDao()::clearAllFeeds
                )
```

re-naming: part of the challange is that this receives 4 methods, only 1 parameter's type is different so `ofFlow` wouldn't look very good imo either :/.